parsers/python.c

```diff
@@ -122,17 +122,17 @@ static roleDesc PythonUnknownRoles [] = {
 };
 
 static kindOption PythonKinds[COUNT_KIND] = {
-	{TRUE, 'c', "class",    "classes"},
-	{TRUE, 'f', "function", "functions"},
-	{TRUE, 'm', "member",   "class members"},
-	{TRUE, 'v', "variable", "variables"},
-	{TRUE, 'I', "namespace", "name referring a module defined in other file"},
-	{TRUE, 'i', "module",    "modules",
-	 .referenceOnly = TRUE,  ATTACH_ROLES(PythonModuleRoles)},
-	{TRUE, 'x', "unknown",   "name referring a classe/variable/function/module defined in other module",
-	 .referenceOnly = FALSE, ATTACH_ROLES(PythonUnknownRoles)},
+	{TRUE, 'c', "class",      "classes"},
+	{TRUE, 'f', "function",   "functions"},
+	{TRUE, 'm', "method",     "class methods"},
```

Comment:
Sounds better, but breaks kinds with the old parser for not much, so I don't know.